javascript/node/selenium-webdriver/lib/webdriver.js

```diff
@@ -1131,6 +1131,14 @@ class Options {
       expiry = Math.floor(date.getTime() / 1000);
     }
 
+    // if not declared, attribute is set to 'undefined'
+    if(typeof sameSite === "undefined") {
+      // do nothing
+    } else if(sameSite!=="Strict" && sameSite!=="Lax") {
```

Comment:
```suggestion
    if(sameSite !== 'Strict' && sameSite !== 'Lax') {
```

not sure about the code style in Selenium but it seems they use single quotes